common/shlibs

```diff
@@ -800,6 +800,7 @@ libsvn_ra_serf-1.so.0 libsvn-1.8.1_1
 libsvn_fs_x-1.so.0 libsvn-1.9.0_1
 libsvn_swig_py-1.so.0 subversion-python-1.7.5_1
 libsvn_swig_perl-1.so.0 subversion-perl-1.7.5_1
+libjson-c.so.5 json-c-0.14.0_1
```

Comment:
```suggestion
libjson-c.so.5 json-c-0.14.0_2
```
and the commit message is misleading. you're revbumping because you added it to shlibs not because of nvme-cli

Reply:
@classabbyamp may I ask you something so that I know what's going on here? The current version of json-c is 0.17 which is already >=0.14.0_[12]. I bumped it to revision 2, as requested. Why do we map libjson-c.so.5 to json-c-0.14.0_2 and not json-c-0.**17**.0_2? Is there any real difference introduced by mapping `libjson-c.so.5 json-c-0.14.0_2` as opposed to `libjson-c.so.5 json-c-0.17.0_2`?

I tried to get my head around it by re-reading MANUAL and meator's guide but still don't quite get it unless it's just a typo.

Reply:
you wrote 14, I don't know why you did it. if the shlib is newly added to common/shlibs, use the current version

Reply:
Yes, I did write 14. I should've asked the question before. But to my defense the Suggested change read what it read. Will fix that in a minute. Thank you for responding.